### DISCUSSION TITLE
"For You" recommends projects you've already starred if many people you follow also interact with that project

### DISCUSSION THREAD

**Discussion by twome** [community/community#27798] [Other Feature Feedback, Questions, & Ideas]
Needless to say, this is a faulty recommendation. Projects that users have already interacted with at all (especially starring) should be added to the blocklist of possible recommendations.

**Comment by jfullstackdev** [comment]
just to confirm this, yes, I'm seeing repos that I have starred on Recommendations filter, hopefully this will be seen by one of the Staff members
